src/pvrclient-nextpvr.cpp

```diff
@@ -800,19 +832,45 @@ bool cPVRClientNextPVR::IsRealTimeStream()
   if (IsServerStreaming())
   {
     if (m_nowPlaying == Recording)
-      return m_recordingBuffer->IsRealTimeStream();
+      return m_multistreamRecording[m_streamCount]->IsRealTimeStream();
```

Comment:
`std::lock_guard<std::recursive_mutex> lock(m_multistreamRecording);` prior to accessing map.

No need to stop the lock, happens automatically at the end block